packages/teleport/src/services/databases/databases.test.ts

```diff
@@ -67,6 +67,24 @@ describe('correct formatting of all type and protocol combos', () => {
   );
 });
 
+test('null labels field in database fetch response', async () => {
+  jest.spyOn(api, 'get').mockResolvedValue([{ labels: null }]);
+
+  const database = new DatabaseService();
+  const response = await database.fetchDatabases('im-a-cluster');
+
+  expect(response[0].tags).toEqual([]);
+});
+
+test('undefined labels field in database fetch response', async () => {
+  jest.spyOn(api, 'get').mockResolvedValue([{ labels: undefined }]);
```

Comment:
we don't need to test for undefined from our api responses. `null` is an assignable value that says "no value" vs `undefined` that says value hasn't been assigned yet: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined#description

Reply:
also: https://stackoverflow.com/a/14946821